### PR TITLE
12.0 mig sale order digitized signature

### PR DESCRIPTION
**Pull request by mgosai**
Depends on:

- [ ] web_widget_digitized_signature: OCA/web#1318

### CODE REVIEW

File: sale_order_digitized_signature/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright (C) 2019 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (nikul-serpentcs) [inline sale_order_digitized_signature/__manifest__.py]: @mgosai Duplicate `License` line, Please Remove it.

File: sale_order_digitized_signature/views/sale_views.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+<odoo>
+
+    <record id="view_order_form_signature" model="ir.ui.view">
+        <field name="name">view_order_form_signature</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='note']" position="after">
+                <div class="oe_clear"/>
+                <label for="customer_signature" class="oe_edit_only"/>
```
Comment (nikul-serpentcs) [inline sale_order_digitized_signature/views/sale_views.xml]: @mgosai IMO Add  label `string`

**Review by nikul-serpentcs — COMMENTED** [review]
Nitpicking

**Comment by max3903** [comment]
Superseeded in #892

**Pull request closed without merging**